metadata.json

```diff
@@ -62,7 +62,8 @@
       "operatingsystem": "Ubuntu",
       "operatingsystemrelease": [
         "16.04",
-        "18.04"
+        "18.04",
+        "20.04"
```

Comment:
can you also add ubuntu 20 to the .sync.yml and ubuntu 20 puppet 6 acceptance tests to the .travis.yml?